src/app/views/datasets-new/edit-metadata/DatasetMetadataController.jsx

```diff
@@ -365,7 +365,10 @@ export class DatasetMetadataController extends Component {
     mapVersionCollectionStateToState = versions => {
         try {
             const version = versions.find(version => {
-                return version.version === this.props.params.versionID && version.edition === this.props.params.editionID;
+                console.log(version)
+                return version.version === this.props.params.versionID && 
```

Comment:
Can `this.props` or `this.props.params` ever be nil?

Reply:
Yes if the user tampers with the URL, but it would have thrown an error (displayed to the user) before getting to this method.

Reply:
👍